dashboard/app/views/levels/show.html.haml

```diff
@@ -34,7 +34,7 @@
     = render partial: 'levels/widget'
   - else
     = render partial: "levels/#{@level.class.to_s.underscore}"
-  - unless @level.is_a?(StandaloneVideo)
+  - unless @level.is_a?(StandaloneVideo) || @level.is_a?(ExternalLink)
```

Comment:
I don't see the need for it. The level has no solution nor related videos nor instructor-specific content.